tests/test_butler.py

```diff
@@ -482,6 +482,26 @@ def testConstructor(self):
         self.assertIsNone(butler2.run)
         self.assertIs(butler.datastore, butler2.datastore)
 
+        # Test that we can use an environment variable to find this
+        # repository.
+        butler_index = Config()
+        butler_index["label"] = self.tmpConfigFile
+        butler_index.dump
```

Comment:
Um, this looks useless.

Reply:
indeed. Left over from a code reorganization.